tests/testsuite.src/run_file.at

```diff
@@ -695,7 +695,7 @@ AT_DATA([prog.cob], [
 ])
 
 AT_CHECK([$COMPILE -fassign-clause=external prog.cob], [0], [], [])
-AT_CAPTURE_FILE(./whatever)
+AT_CAPTURE_FILE([./whatever])
```

Comment:
Those captures are a good catch. I've adjusted them (no need for the dot either):
AT_CAPTURE_FILE(./something) ->AT_CAPTURE_FILE([something])
AT_CAPTURE_FILE(SOME) -> AT_CAPTURE_FILE([SOME])